### ISSUE TITLE
Provide plugin for HDF Server resources

### ISSUE THREAD

**Issue by jreadey** [HDFGroup/hdf-compass#78]
Create HDFServer client plugin.

**Comment by giumas** [comment]
@jreadey Is this something that you are working on?

**Comment by jreadey** [comment]
@giumas - Yes, it makes sense for me to tackle this.

To avoid stepping on each other's toes, let's all follow this convention:

Assign the issue to yourself if you intend to start working on it in the next day or so.  If things change and you don't have time for it, just set back to unassigned - no harm done.

**Comment by jreadey** [comment]
This was implemented in PR #138 and subsequently merged into develop branch.

Some limitations with this implementation: Compound types and soft/external links are not supported.

I'll revisit once the h5pyd (see h5serv project) package is ready for release.
